src/stack/lib/block-stack.scss

```diff
@@ -0,0 +1,26 @@
+.tbds-block-stack {
+  --gap: #{$tbds-space-2};
+}
+
+.tbds-block-stack--gap-2x {
+  --gap: #{$tbds-space-3};
+}
+
+.tbds-block-stack--gap-3x {
+  --gap: #{$tbds-space-4};
+}
+
+.tbds-block-stack--gap-4x {
+  --gap: #{$tbds-space-5};
+}
+
+.tbds-block-stack__item:not(:last-child) {
+  margin-bottom: calc(var(--gap) / 2);
+  padding-bottom: calc(var(--gap) / 2);
+}
+
+.tbds-block-stack--bordered {
+  .tbds-block-stack__item:not(:last-child) {
+    border-bottom: 1px solid #bbb;
```

Comment:
This (and other color values) might be easier to refactor later if it's a private sass variable for now.

Reply:
Yeah, that’s a good thought. We do eventually need to systemize borders.

Reply:
Resolved in 8c3489792da034972610f0a6312f432ef6bdfcae